src/components/LoginPage.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+const LoginPage = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const history = useHistory();
+
+  const handleLogin = (event) => {
+    event.preventDefault();
+
+    // For simplicity, assume these are the correct credentials
+    if (username === 'student' && password === 'password') {
+      history.push('/dashboard');
+    } else {
+      alert('Invalid credentials');
+    }
+  }
+
+  return (
```

Comment:
Using alert for error handling is not a good user experience. Consider using a more user-friendly way to display errors, such as inline error messages or a toast notification.